jabkit/src/main/java/module-info.java

```diff
@@ -22,7 +23,6 @@
     requires org.tinylog.impl;
 
     requires java.xml;
-  
```

Comment:
Please do not modify lines of code not related to the PR if not 90% sure

Here, the empty line does make sence to separate blocks. Please undo this change.